packages/scripts/CHANGELOG.md

```diff
@@ -4,235 +4,238 @@
 
 ### Breaking Changes
 
-- The default babel configuration has changed to only support stage-4 proposals. This affects the `build` and `start` commands that use the bundled babel configuration; if a project provides its own, this change doesn't affect it. [#22083](https://github.com/WordPress/gutenberg/pull/22083)
-- The bundled `wp-prettier` dependency has been upgraded from `1.19.1` to `2.0.5`. Refer to the [Prettier 2.0 "2020" blog post](https://prettier.io/blog/2020/03/21/2.0.0.html) for full details about the major changes included in Prettier 2.0.
-- The bundled `eslint` dependency has been updated from requiring `^6.8.0` to requiring `^7.1.0`.
+-   The `env` family of scripts has been removed.
```

Comment:
It would be great to include detailed info about the alternative so people don’t miss the opportunity to migrate. A link to this PR might be also helpful for those who want to explore this change further.

Reply:
Updated